test/net-to-gross/calcTaxableIncome.test.js

```diff
@@ -0,0 +1,121 @@
+const assert = require('assert');
+const test = require('mocha').it;
+const calcTaxableIncome = require('../../src/net-to-gross/calcTaxableIncome.js');
+
+describe('Calculate Taxable Income', function () {
+
+    describe('Test Taxable Income when no dependent deducted', function () {
+
+        test('No taxable income when Net Income is lte 11_000_000', async function () {
+            const netIncome = 11_000_000
+            const actual = await calcTaxableIncome(netIncome)
+            const expected = 0;
+
+            assert.equal(actual, expected)
+        })
+
+        test('The total tax is calculated correctly for tax rate 1 only', async function () {
+            // ti                   <= net + 5% * Min(ti, 5_000_000) - 11_000_000; Only if ti <= 5_000_000
```

Comment:
Pls remove these comment since you already have a very explaination in `src/net-to-gross/calcTaxableIncome.js`.

Reply:
You can reference the file as a code comment.